src/commons/BrowserUtils.ts

```diff
@@ -757,6 +774,17 @@ export namespace BrowserUtils {
     }
   }
 
+  /**
+   *
+   * @param selector - element for get size
+   */
+  export function getElementSize(selector: string): SizeReturn {
+    Reporter.debug(`Get Element: '${selector}' size`);
+    const element: Element<void> = $(selector);
```

Comment:
add isExist(selector)
so it will wait for element to exist or fail with error

Reply:
done